plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -557,8 +558,12 @@ private void testSelectOrPartitionedByTimestampWithTimeZone(boolean partitioned)
                             "(NULL, NULL, NULL, NULL, 1e0, NULL, NULL)");
         }
         else {
+            // Disable default filter factor to avoid getting stats when NDV is not present
```

Comment:
cc @findepi @losipiuk.
This makes me wonder if we shouldn't have two default filters factor triggers
* one global
* one for individual predicates that we cannot estimate.